lib/Service.php

```diff
@@ -241,6 +241,7 @@ public function write(string $rootElementName, $value, string $contextUri = null
     public function mapValueObject(string $elementName, string $className): void
     {
         list($namespace) = self::parseClarkNotation($elementName);
+        $namespace = $namespace ?? '';
```

Comment:
`parseClarkNotation` says that it can return `string|null` in the first element of the returned array.
later uses of `$namespace` expect a `string`. So just convert `null` to the empty string here.

Same for a few place in Writer.php below.